front/src/app/components/common/toggler/index.tsx

```diff
@@ -18,6 +18,7 @@ export interface ITogglerProps {
     className?: string;
     cssClasses?: ICssClasses;
     onChange?: (params: ITogglerChangeParams) => void;
+    type?: string;
```

Comment:
please use evident type def for this props:
type?: 'checkbox' | 'chotuteshemozhetbit' | 'smth'

Reply:
'checkbox' | 'radiobutton' i think